base-pi/py_server.py

```
@@ -5,15 +5,37 @@
 import signal
 from metrics import asyncsshloop, register_metric_events, cpuloop, send_fake_antenna_stats
 from shutdown import register_shutdown_commands
-import sys
+import sys, subprocess
+
+
+print("\033[0m----------------")
+
+# Get commit hash and message
+commit_result = subprocess.run(['git', 'log', '-1', '--pretty=format:%h %s'], capture_output=True, text=True)
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)